Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,25 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250])
+    var
+        ConstUpgradeTags: Record "Upgrade Tags";
+    begin
+        ConstUpgradeTags.Reset;
+        ConstUpgradeTags.SetRange(Tag, ExistingTag);
+        ConstUpgradeTags.ModifyAll(ConstUpgradeTags."Skipped Upgrade", true);
+    end;
+
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; TagCompanyName: Code[30])
+    var
+        ConstUpgradeTags: Record "Upgrade Tags";
+    begin
+        ConstUpgradeTags.Reset;
```

Comment:
Same comment here - Const should be removed.

Reply:
Removed Const